qabel_provider/views.py

```diff
@@ -81,6 +82,41 @@ def auth_resource(request, format=None):
     })
 
 
+@api_view(('POST',))
+def info_resource(request, format=None):
+    """
+    Return user information by user ID.
+
+    Provides access to user information for other services like qabel-block.
+
+    :param request: rest request
+    :param format: ignored, because the resource never responds with a body
```

Comment:
Indeed. I will fix the docs here and the doc of auth_resource as well (copy-pasted).